internal/db/schema/migrations/oss/postgres/29/02_kms.up.sql

```diff
@@ -0,0 +1,22 @@
+-- TODO (jimlambrt 4/2002) this additional migration needs to be in a future PR.
```

Comment:
i would not include this file, the migration system will "run" it and include the file number in the schema version that is recorded. Instead I would include these details in #2026